Makefile

```diff
@@ -7,8 +7,8 @@
 
 BUILD_DIR = OBJROOT="$(CURDIR)/build" SYMROOT="$(CURDIR)/build"
 SHELL = /bin/bash -e -o pipefail
-IOS32 = -scheme OCMockLib -sdk iphonesimulator -destination 'name=iPhone 5' $(BUILD_DIR)
-IOS64 = -scheme OCMockLib -sdk iphonesimulator -destination 'name=iPhone 5s' $(BUILD_DIR)
+IOS32 = -scheme OCMockLib -destination 'platform=iOS Simulator,OS=10.3.1,name=iPhone 5' $(BUILD_DIR)
```

Comment:
Because iOS 11 dropped the 32-bit architectures (armv7 & armv7s), you have to specify the OS so that the iPhone 5 can run. The latest simulator OS that can run 32-bit architectures happens to be 10.3.1 included in Xcode 10+.

Reply:
Of course! Totally forgot about that.